bin/run.sh

```diff
@@ -0,0 +1,13 @@
+docker run \
+    --rm \
+    -u `id -u`:`id -g` \
+    --name dm_forwarder \
+    --network host \
+    --mount type=volume,src=dm_forwarder,target=/var/tmp/data \
+    --mount type=volume,src=dm_forwarder,target=/var/log/iip \
+    --mount type=bind,src=/etc/passwd,target=/etc/passwd,readonly \
```

Comment:
Is mounting /etc/passwd necessary?

Reply:
Yes. when `docker run` is executed using `uid:gid`, inside the container, ssh cannot copy the files because `uid` does not exist in the `/etc/passwd` inside container and doesn't allow copying of files.